include/sb_handle/portblas_handle.h

```diff
@@ -174,21 +174,21 @@ class SB_Handle {
 
   inline void wait() { q_.wait(); }
 
-  inline void wait(std::vector<cl::sycl::event> evs) {
-    cl::sycl::event::wait(evs);
+  inline void wait(std::vector<sycl::event> evs) {
+    sycl::event::wait(evs);
   }
 
-  inline void wait(cl::sycl::event ev) { cl::sycl::event::wait({ev}); }
+  inline void wait(sycl::event ev) { sycl::event::wait({ev}); }
 
   /*  @brief waiting for a list of sycl events
- @param first_event  and next_events are instances of sycl::sycl::event
```

Comment:
Garbage in, garbage out!

Reply:
Fixed in https://github.com/codeplaysoftware/portBLAS/pull/523#discussion_r1627979553 / https://github.com/codeplaysoftware/portBLAS/commit/bf873cad0a580c2aac4f67970ac987de0d93f1dc.